utilities/_utilities.clears.scss

```diff
@@ -1,3 +1,4 @@
+//TODO: Make these into Mixins
```

Comment:
`// TODO: Convert into mixins`

Just cleaning up formatting to match `images.scss` todo

Reply:
done